frontend/www/js/omegaup/components/arena/ContestPractice.vue

```diff
@@ -3,35 +3,46 @@
     <div class="title">
       <h1>
         <span>{{ contest.title }}</span>
-        <sup class="socket-status" title="WebSocket">•</sup>
+        <sup class="socket-status" title="WebSocket">✗</sup>
       </h1>
       <div class="clock">∞</div>
     </div>
     <div class="tab navleft">
       <div class="navbar">
         <omegaup-arena-navbar-problems
           :problems="problems"
-          :active-problem="activeProblem"
+          :active-problem="activeProblem.alias"
           :in-assignment="false"
           :digits-after-decimal-point="contest.partialScore ? 2 : 0"
-          @disable-active-problem="activeProblem = null"
+          @disable-active-problem="activeProblem.alias = null"
           @navigate-to-problem="onNavigateToProblem"
         ></omegaup-arena-navbar-problems>
       </div>
       <omegaup-arena-contest-summary
-        v-if="activeProblem === null"
+        v-if="activeProblem.alias === null"
         :contest="contest"
         :show-ranking="false"
       ></omegaup-arena-contest-summary>
       <div v-else class="problem main">
         <omegaup-problem-details
           :user="{ loggedIn: true, admin: false, reviewer: false }"
-          :problem="problem"
+          :problem="problemInfo"
           :active-tab="'problems'"
-          :runs="runs"
+          :runs="activeProblem.runs"
+          :should-show-clarifications="true"
         >
-          <template #quality-nomination-buttons></template>
-          <template #best-solvers-list></template>
+          <template #quality-nomination-buttons><div></div></template>
```

Comment:
```suggestion
          <template #quality-nomination-buttons></template>
```

qué pasa si quitamos este `<div>` vacío? igual abajo.

Reply:
Si le quito esos `<div>` se muestra la información original del `slot`. Intenté colocar una etiqueta `<template>`, pero no funcionó.